### ISSUE TITLE
Active sensor.heating_notifications leads to error

### ISSUE THREAD

**Issue by fuslwusl** [bosch-thermostat/home-assistant-bosch-custom-component#85]
**Describe the bug**
When the entity sensor.heating_notifications is active the integration stops working and many sensor values from the heating have value -1.

**To Reproduce**
Activate entity sensor.heating_notifications and when the heating sends results you get an error in the logfile.

```
homeassistant    | 2021-09-03 12:05:03 DEBUG (MainThread) [bosch_thermostat_client.connectors.http] Sending GET request to /notifications
homeassistant    | 2021-09-03 12:05:03 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
homeassistant    | Traceback (most recent call last):
homeassistant    |   File "/config/custom_components/bosch/__init__.py", line 377, in thermostat_refresh
homeassistant    |     await self.component_update(SENSOR, event_time)
homeassistant    |   File "/config/custom_components/bosch/__init__.py", line 356, in component_update
homeassistant    |     await entity.bosch_object.update()
homeassistant    |   File "/usr/local/lib/python3.9/site-packages/bosch_thermostat_client/helper.py", line 252, in update
homeassistant    |     self.process_results(result, key)
homeassistant    |   File "/usr/local/lib/python3.9/site-packages/bosch_thermostat_client/sensors/__init__.py", line 270, in process_results
homeassistant    |     data[RESULT][VALUE + key_suffix] = self.errorcodes[ccd]["title"]
homeassistant    | KeyError: '227'
```
When i disable this entity the requests are fine and there is no error and i get all sensor values.

**Version**
  - 2021.8
  - HA Bosch component latest
  - Python Bosch HTTP client version?
  - IVT von RC300 RC300/RC310/Nefit Moduline 3000 Firmware: 04.07.07

** Debug SCAN **
https://jsonblob.com/883296294244990976

**Comment by pszafer** [comment]
I will fix it.  Do you know what your error means?

**Comment by fuslwusl** [comment]
6A 277 is some kind of ionization error - current too low.

Cause
Flame monitoring

Description
No ionization message after ignition. After the fourth attempt, this message is displayed.

**Comment by pszafer** [comment]
Should be fixed in prerelease 0.14.0.
